docs/references/javascript/types/paginated-request.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: PaginatedRequest
+description: An interface that describes the request params of a method that returns a paginated list of resources.
+---
+
+# `PaginatedRequest<T>`
+
+An interface that describes the request params of a method that returns a paginated list of resources.
+
+## Properties
+
+| Properties | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | `number` |  A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. Defaults to `1`.|
```

Comment:
```suggestion
| `initialPage?` | `number` |  A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it will skip the first 9 pages and will fetch the 10th page. Defaults to `1`.|
```